projects/dotcms/src/lib/api/DotApiPage.ts

```diff
@@ -3,6 +3,11 @@ import { DotCMSHttpClient } from '../utils/DotCMSHttpClient';
 import { DotCMSPageAsset, DotCMSError } from '../models';
 import { DotApiLanguage } from './DotApiLanguage';
 
+export const enum DotCMSPageFormat {
```

Comment:
Can you please move this enum to it own file inside: `projects/dotcms/src/lib/models`

Reply:
Done!